cfgov/unprocessed/apps/mmt-cost-of-credit-tool/js/views/home/index.js

```diff
@@ -0,0 +1,51 @@
+import { useCallback } from "react";
+import { useHistory, Link } from "react-router-dom";
+import Hero from "../../components/hero";
+import Button from "../../components/button";
+import { useScrollToTop } from "../../components/scroll-to-top";
+
+import heroImg from "img/Hero_2.png";
+import arrowRight from "@cfpb/cfpb-icons/src/icons/arrow-right.svg";
+
+export default function Home() {
+  useScrollToTop();
+
+  const history = useHistory();
+
+  const nextPage = useCallback(
+    evt => {
+      evt.preventDefault();
+      history.push("/tool");
+    },
+    [history]
+  );
+
+  // This is the Home Page of the app
+  return (
+    <main className="coc-view home">
+      <Hero
+        tag="True Cost Tool"
+        title="Should I use credit to buy this?"
+        subtitle="Are you considering making a purchase using credit to borrow money?  This tool will help
```

Comment:
The length of the text here makes me think we should get rid of the `subtitle` prop in the `Hero` component, and instead just make it render `props.children` in the subtitle area.